website/content/docs/bloop-basics.md

```diff
@@ -24,23 +24,24 @@ of our build tool, and `bloop` aims to address them.
 
 We have created bloop to make you productive without getting in your way. We
```

Comment:
I explain that use case a little bit after, on the "Not a build tool" section. Is it clear?

Reply:
I just prefer not to explain it here upfront since I think that's use case is a little bit niche, and the docs are mainly targeting developrs that use bloop to compile/test their code.